### ISSUE TITLE
"Can't find preference path for IntelliJIdea" error

### ISSUE THREAD

**Issue by h-marvin** [bchatard/alfred-jetbrains#30]
Q | A
--- | ---
Workflow version | _1.0.12_
OSX Version | _10.14.5_
Alfred Version | _3.8.2_
JetBrains Product/Version | _IntelliJ IDEA CE / 2019.1.3_
Installation type | _classic .dmg_

### Description
Getting the following error on usage:
```
Error: Can't find preference path for IntelliJIdea
    at getPreferencePath (/usr/local/lib/node_modules/@bchatard/alfred-jetbrains/src/product.js:77:9)
    at Object.get (/usr/local/lib/node_modules/@bchatard/alfred-jetbrains/src/product.js:117:30)
    at Object.<anonymous> (/usr/local/lib/node_modules/@bchatard/alfred-jetbrains/src/index.js:15:38)
```

-
JetBrains - Open Project - v3 1.0.12
Alfred 3.8.2
darwin x64 18.6.0

### Step to reproduce
- installation via `npm install -g @bchatard/alfred-jetbrains`
- invoke Alfred
- type `idea`

Probably there are other steps necessary to reproduce, but that's all I did :/

**Comment by bchatard** [comment]
Hi,

By default this workflow works with "Ultimate Edition", so you need to configure the workflow to use CE  *preferences path*. Please check the [doc](https://github.com/bchatard/alfred-jetbrains/blob/master/doc/customisation/product.md). 
Do not hesitate to ask more information if the documentation is not clear.

Your issue is a good exemple of my poor documentation 😟 thanks 😀

**Comment by h-marvin** [comment]
That worked, thanks.

For anyone else reading this issue:

I created `~/.config/alfred-jetbrains/custom.json` with this content:
```
{
  "IdeaCE": {
    "preferences": "IdeaIC",
    "bin": "idea"
  },
  "IdeaIE": {
    "preferences": "IdeaIC",
    "bin": "idea"
  },
  "IdeaUE": {
    "preferences": "IdeaIC",
    "bin": "idea"
  }
}
```

Additionally I added `/.config/alfred-jetbrains/custom.json` as value to env variable `jb_product_customisation_file` (as described in the doc linked by @bchatard above).

Thanks so much. What an awesome workflow to have now 🖤
